proxy/src/main.rs

```diff
@@ -1,8 +1,16 @@
 //! Proxy to serve a specialised HTTP to the oscoin MVP.
 
-#![deny(clippy::all, clippy::pedantic)]
 #![deny(missing_docs)]
 #![deny(warnings)]
+#![forbid(
+    clippy::all,
+    // clippy::cargo,
```

Comment:
Does it make sense to commit commented out code into the repo?

Reply:
Good point, tried out hardmode with all clippy checks. But the `cargo` one only makes sense for public crates consumed as libraries.